validation/utils.go

```diff
@@ -0,0 +1,27 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package validation
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ValidateTokenAndServerURL(token string, serverURL string) {
```

Comment:
Will it make sense to validate each separately, i.e. to have two functions?

(Otherwise we may rename the function, because "validate token and server url" may read ambiguously. Perhaps `validateServerUrlAndAccessToken()`.)

Reply:
yeah, breaking it into separate functions make sense, especially since serverURL could later be validated in the swagger client before calling the api